net/local/local_netpoll.c

```diff
@@ -125,21 +126,7 @@ void local_event_pollnotify(FAR struct local_conn_s *conn,
                             pollevent_t eventset)
 {
 #ifdef CONFIG_NET_LOCAL_STREAM
-  int i;
-
-  for (i = 0; i < LOCAL_NPOLLWAITERS; i++)
-    {
-      struct pollfd *fds = conn->lc_event_fds[i];
-      if (fds)
-        {
-          fds->revents |= (fds->events & eventset);
-          if (fds->revents != 0)
-            {
-              ninfo("Report events: %08" PRIx32 "\n", fds->revents);
-              nxsem_post(fds->sem);
-            }
-        }
-    }
+  poll_notify(conn->lc_event_fds, LOCAL_NPOLLWAITERS, eventset);
```

Comment:
remove local_event_pollnotify

Reply:
Several places call local_event_pollnotify(), seems leave it here is better.